output/schema.json

```diff
@@ -89,8 +70,15 @@
       },
       "required": [ "valid" ]
     },
-    "basic": { "$ref": "#/$defs/outputUnit" },
-    "detailed": { "$ref": "#/$defs/outputUnit" },
-    "verbose": { "$ref": "#/$defs/outputUnit" }
+    "basic": {
+      "properties": {
+        "valid": { "type": "boolean" },
+        "nested": {
+          "$ref": "#/$defs/outputUnitArray"
+        }
+      },
+      "required": [ "valid", "nested" ]
```

Comment:
Why does the 'basic' format contain 'nested'? My understanding was that basic is still a flat structure.

Reply:
Basic is an object with only `valid` and `nested ` at the root.  The rest is normal output units in the `nested` array.

```json
{
  "valid": true,
  "nested": [ <flat list> ]
}
```

This provides the overall validation result at the root and a flat list.  It's the same as 2020-12, except that the location properties aren't required at the root (because they're always just `/` and not necessary).